package.json

```diff
@@ -73,6 +76,12 @@
     "test": "mocha --full-trace --require @babel/register test/geotiff.spec.js"
   },
   "author": "Fabian Schindler",
+  "browser": {
+    "fs": false,
+    "http": false,
+    "https": false,
+    "url": false
```

Comment:
These imports are used in `src/source/file.js` which will not be called in the browser environment.